### DISCUSSION TITLE
Guide on running trucky

### DISCUSSION THREAD

**Discussion by ibndiaye** [truckersmp-cli/truckersmp-cli#318] [Q&A] [answered]
Is there a guide on how to run trucky with TMP-cli? I cant seem to find the .exe that i can put in my config
Any help would be appreciated

**Comment by Lucki** [comment]
> Is there a guide on how to run trucky with TMP-cli?

No. At least not yet ;)

> I cant seem to find the .exe that i can put in my config

Well, here's some general advice:

1. Aquire `program`.
2. Do steps required to get `program` running.
  This usually involves one or more of the following steps:
    * Decompress (unzip) `program`.
      If this is the only step:
      * Decompress into a premanent folder you're happy with.
      * Make sure to remember the decompressed `path`.
    * Install `program`.
      Make sure to remember the installation `path`.
      Make sure you execute the installation in the [correct prefix](https://github.com/truckersmp-cli/truckersmp-cli#wineprefix).
      Example: `WINEPREFIX="/path/to/prefix" wine "/path/to/program"`
    * Install required dependencies for `program`.
      Make sure you install the dependencies in the [correct prefix](https://github.com/truckersmp-cli/truckersmp-cli#wineprefix).
      Example: `WINEPREFIX="/path/to/prefix" wine "/path/to/dependency"`
      Winetricks can be a great help here for standard dependencies like `dotnet40` and alike.
      Example: `WINEPREFIX="/path/to/prefix" winetricks dotnet40`
1. Find the `executable_path` in the remembered `path`.
1. Adjust the [configuration file](https://github.com/truckersmp-cli/truckersmp-cli#configuration-file) according to the `executable_path` result from the previous step.

**Reply by sodlier** [reply]
how do we aquire program

**Comment by kallestrop** [comment, marked as answer]
Trucky has released a native Linux build of their tracking software which doesn't need Overwolf anymore.

It works with Proton game version at the moment, maybe later there will be a Trucky client for the native Linux version.

https://truckyapp.com/blog/announcements/trucky-for-linux-is-now-available/

**Reply by Lucki** [reply]
Great find! With this it should now be as easy as starting the Trucky AppImage first and truckersmp-cli afterward.

Direct help link: https://truckyapp.com/kb/linux-client/
